Swift/BRCrypto/common/BRCryptoSigner.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import BRCryptoC
```

Comment:
Is it a future goal to have this in C as well?

Reply:
Oh, maybe.  Everything in common/ that touchs bitcoin needs to be in C, it seems?

Reply:
That would be ideal, from my POV, so that we don't have parallel implementations.